README.md

```diff
@@ -1,73 +1,120 @@
-
-[travis-badge]: https://api.travis-ci.org/tarantool/tarantool.svg?branch=master
-[travis-url]: https://travis-ci.org/tarantool/tarantool
-[gitlab-ci-badge]: https://gitlab.com/tarantool/tarantool/badges/master/pipeline.svg
-[gitlab-ci-url]: https://gitlab.com/tarantool/tarantool/commits/master
-[coverage-badge]: https://coveralls.io/repos/github/tarantool/tarantool/badge.svg?branch=master
-[coverage-url]: https://coveralls.io/github/tarantool/tarantool?branch=master
-[groups-badge]: https://img.shields.io/badge/Google-Groups-orange.svg
-[groups-url]: https://groups.google.com/forum/#!forum/tarantool
-[telegram-badge]: https://img.shields.io/badge/Telegram-join%20chat-blue.svg
-[telegram-url]: http://telegram.me/tarantool
-[slack-badge]: https://img.shields.io/badge/Slack-join%20chat-lightgrey.svg
-[slack-url]: http://slack.tarantool.org/
-[gitter-badge]: https://badges.gitter.im/Join%20Chat.svg
-[gitter-url]: https://gitter.im/tarantool/tarantool
+<p align="left">
+  <a href="https://tarantool.io/en/">
+    <img
+      alt="Tarantool"
+      src="https://i.imgur.com/BUYqImx.png"
+      width="400"
+    />
+  </a>
+</p>
+
+**A feature-rich in-memory database.**  It has the performance of a key-value
+cache with the functionality you would expect from a regular database:
+
+* SQL
+* ACID
+* primary and secondary indexes
+* disk engine
+* stored procedures
+* replication
+* sharding
+* and a wealth of modules and connectors from popular languages
+
+Another Tarantool feature is the ability to write code in Lua, a general purpose
+programming language, right inside the database. You can write custom expiration
+strategies and any business logic, and synchronize data with other data sources.
+
+You can use Tarantool for any KV purposes -- or use it to replace Redis when
+you need secondary indexes, a different structure of storage data, SQL, etc.
+
+For more details, please see the
+[Tarantool documentation](https://www.tarantool.io/en/doc/2.2/).
+
+## Getting started
+
+### Get hold of Tarantool
+
+* Try online:
+
+  Visit our playground at https://www.tarantool.io/en/try-dev/
+
+* Use a Docker image:
+
+  ```bash
+  docker run -d -p 3301:3301 -v $(pwd):/var/lib/tarantool tarantool/tarantool:2
+  ```
+
+* Install locally:
+
+  * On Ubuntu/Debian/Centos:
+
+    ```bash
+    curl -L http://tarantool.io/installer.sh | VER=2.2 sudo bash
+    ```
+
+  * On Mac OS using [Homebrew](https://brew.sh):
+
+    ```bash
+    brew install tarantool
+    ```
+
+  For more details, please visit https://tarantool.io/download.
+
+* Build from source:
+
+  Please see
+  [detailed instructions](https://tarantool.io/doc/2.2/dev_guide/building_from_source/)
+  in the Tarantool documentation.
+
+### Connectors
+
+We have connectors for many popular languages:
+
+* [Python](https://github.com/tarantool/tarantool-python)
+* [PHP](https://github.com/tarantool-php/client)
+* [Java](http://github.com/tarantool/tarantool-java/)
+* [Go](https://github.com/mialinx/go-tarantool)
```

Comment:
I have no strict approach:

* Python: Why official one, not igorcoding/asynctnt that supports asynchronous connections?
* PHP: Why unofficial? There is official one (it is however near to be in unmaintained state).

Let's use best ones: igorcoding/asynctnt and tarantool-php/client.

The link you provided for go-tarantool now redirects to https://github.com/tarantool/go-tarantool

Reply:
There are also lomik/go-tarantool, which may be considered better in some ways

Reply:
And why not https://github.com/tarantool/go-tarantool after all

Reply:
It was not resolved.